README.md

```diff
@@ -61,10 +61,9 @@ _or_
 - [padding-around-before-all-blocks](docs/rules/padding-around-before-all-blocks.md)
 - [padding-around-before-each-blocks](docs/rules/padding-around-before-each-blocks.md)
 - [padding-around-expect-statements](docs/rules/padding-around-expect-statements.md)
+- [padding-around-describe-blocks](docs/rules/padding-around-describe-blocks.md)
 
-- [padding-before-describe-blocks](docs/rules/padding-before-describe-blocks.md)
 - [padding-before-test-blocks](docs/rules/padding-before-test-blocks.md)
-
```

Comment:
Just a little blank line clean up here. Markdown was formatting the main list with extra space.